archivessnake/unpublish_processing_note.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import os
+import json
+import time
+from asnake.aspace import ASpace
+
+
+aspace = ASpace(
+              baseurl='',
+              user='',
+              password=''
+    )
+
+repo = aspace.repositories(2)
+start_time = time.time()
+
+
+def get_resources_notes():
+    for object_type in ['resources']:
+        for object in getattr(aspace, object_type):
+            if (object.jsonmodel_type == 'resource' and object.id_0.startswith(('FA'))):
+                unpublish_notes(object)
+
+def get_ao_notes():
+    for object in repo.archival_objects:
+        unpublish_notes(object)
+
+def unpublish_notes(object):
+    object_json = object.json()
+    for idx, note in list(enumerate(object_json.get('notes'))):
```

Comment:
I believe you can just do 
```suggestion
    for idx, note in enumerate(object_json.get('notes')):
```
since my understanding is that enumerate returns an iterable. 

Let me know if that doesn't work!